tests/containers/podman_image.pm

```diff
@@ -30,6 +30,7 @@ sub run {
             record_info "IMAGE", "Testing image: $iname";
             test_container_image(image => $iname, runtime => $engine);
             test_rpm_db_backend(image => $iname, runtime => $engine);
+            test_systemd_install(image => $iname, runtime => $engine);
```

Comment:
let's keep it here as it's a similar check than rpm_db, but I might move these 2 things to podman.pm/docker.pm as it's more a host test than image test.

Reply:
I would vote for keeping it as an image test rather than host.